src/__tests__/commands/flex/plugins/start.test.ts

```diff
@@ -11,13 +15,61 @@ describe('Commands/FlexPluginsStart', () => {
   start()
     .setup((instance) => {
       sinon.stub(instance, 'runScript').returnsThis();
+      sinon.stub(instance, 'isPluginFolder').returns(true);
+      sinon.stub(instance, 'pkg').get(() => pkg);
     })
     .test(async (instance) => {
       await instance.doRun();
 
       expect(instance.runScript).to.have.been.calledTwice;
-      expect(instance.runScript).to.have.been.calledWith('check-start');
-      expect(instance.runScript).to.have.been.calledWith('start');
+      expect(instance.runScript).to.have.been.calledWith('start', ['flex', '--name', 'plugin-test']);
+      expect(instance.runScript).to.have.been.calledWith('start', ['plugin', '--name', 'plugin-test']);
     })
-    .it('should run start script');
+    .it('should run start script for the directory plugin');
+
+  start(['start', '--name', 'plugin-testOne', '--name', 'plugin-testTwo', '--include-remote'])
```

Comment:
Why are all of these tests taking `start` arg? That's actually then making `twilio flex:plugins:start start --name ...`